README.md

```diff
@@ -24,7 +25,9 @@ docker run -d --name meteor-apm-server \
 
 This can be useful for running an instance quickly in your own environment with orchestration.
 
-### Setup steps
+### Setup steps using MUP
+
+Keep in mind when using MUP: [your user account on the server must be sudoer without password](http://meteor-up.com/docs.html#ssh-based-authentication-with-sudo)
```

Comment:
Why is this necessary? It works for me without.

Reply:
the links give the details about this. maybe your username is root ?

Reply:
no, but doesn't matter. this comment won't hurt :)